assets/js/modules/surveys.js

```diff
@@ -60,6 +73,10 @@ const showSurvey = survey => {
         methods: {
             toggleSurvey: function() {
                 this.isActivated = !this.isActivated;
+
+                if (hasiOSBug()) {
+                    $('body').toggleClass('is-ios-editing-survey');
```

Comment:
Does this mean that on iOS, anyone who activates the survey can no longer scroll? Might be nicer if this class only gets toggled when they focus on the textbox?